src/core_plugins/kibana/server/tutorials/ceph_metrics/index.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { TUTORIAL_CATEGORY } from '../../../common/tutorials/tutorial_category';
+import { onPremInstructions, cloudInstructions, onPremCloudInstructions } from '../../../common/tutorials/metricbeat_instructions';
+
+export function cephMetricsSpecProvider() {
+  const moduleName = 'ceph';
+  return {
+    id: 'cephMetrics',
+    name: 'Ceph metrics',
+    isBeta: true,
+    category: TUTORIAL_CATEGORY.METRICS,
+    shortDescription: 'Fetch internal metrics from the Ceph server.',
+    longDescription: 'The `ceph` Metricbeat module fetches internal metrics from Ceph.' +
+                     ' [Learn more]({config.docs.beats.metricbeat}/metricbeat-module-ceph.html).',
+    artifacts: {
+      application: {
+        label: 'Discover',
+        path: '/app/kibana#/discover'
```

Comment:
@tsg @alexfrancoeur I'm jumping here now to the discovery page. I was thinking if we should jump to discovery and already filter for the data from the module?

Reply:
Another option is to leave that out and then there is simply no button at the end. But I think Discover is not bad.

Reply:
I tried this at first but it feels awkward as the user kind of gets stuck. Everything is complete, what now? Will move forward with discover for now and can still improve it in the future.